cmd/tcpgoon.go

```diff
@@ -56,7 +59,15 @@ func validateRequiredArgs(params *tcpgoonParams, args []string) error {
 	if len(args) != 2 {
 		return errors.New("Number of required parameters doesn't match")
 	}
-	params.hostPtr = args[0]
+	params.hostName = args[0]
```

Comment:
as hostName may not be a hostname (but an IP address), I'd consider this as "target"

Reply:
and i would put, in any case, hostname and not hostName, wouldnt you?